addons/stock/models/stock_traceability.py

```diff
@@ -112,7 +112,10 @@ def _get_usage(self, move_line):
     def _make_dict_move(self, level, parent_id, move_line, unfoldable=False):
         res_model, res_id, ref = self._get_reference(move_line)
         dummy, is_used = self._get_linked_move_lines(move_line)
-        unfoldable = False if not move_line.lot_id else unfoldable
+        if level == 1:
```

Comment:
this doesnt work, create a reception picking, receipt the a product tracked validate open the traceability report from picking, you've got a line unfoldable that show the same line. maybe what you want is unfoldable= unfoldable (the args) you have to check.